controllers/controller_test.go

```diff
@@ -97,6 +97,158 @@ var _ = Describe("Network Controller", Label("network"), Ordered, func() {
 
 			Expect(updatedNetwork.Spec.PeeredIDs).To(Equal([]int32{4, 5}))
 		})
+		It("should update peering status successfully", func() {
+			By("creating a network")
+			network = &metalnetv1alpha1.Network{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-network",
+					Namespace: ns.Name,
+				},
+				Spec: metalnetv1alpha1.NetworkSpec{
+					ID:        111,
+					PeeredIDs: []int32{112},
+				},
+			}
+			Expect(k8sClient.Create(ctx, network)).To(Succeed())
+
+			By("ensuring network is created")
+			Eventually(Get(network)).Should(Succeed())
+
+			Expect(networkReconcile(ctx, *network)).To(Succeed())
+
+			By("defining NetworkInterface object to use created network")
```

Comment:
Do we really need to create a `NetworkInterface` for the peering test?

Reply:
Yes we need it, this is the reply i had got from guvence: We need to create at least one NetworkInterface or LoadBalancer in the network/VNI then only ownVniAvail becomes true. If there is no object in the Network/VNI on that dpservice then there is nothing to peer.